src/arrayMethodSort.js

```diff
@@ -5,7 +5,36 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    let toRepeat = true;
+
+    const bubbleSort = () => {
+      const defaultCheck = (a, b) => {
```

Comment:
You can move to separated variable this `defaultCheck` function (service function) to make your code clear